xpath-outcome-initiator/src/main/java/org/cristalise/kernel/persistency/outcomebuilder/field/StringField.java

```diff
@@ -327,6 +327,9 @@ protected void setAppInfoDynamicFormsJsonValue(AnyNode node, JSONObject json) {
             else if (name.equals("errmsg")) {
                 errmsg = value;
             }
+            else if (name.equals("container")) {
+                ngcontainer = value;
+            }

```

Comment:
Add unit test both to dsl and to outcome builder.

Reply:
will add this to unit test.

Reply:
Can you provide the wiki link so that I know how to confîgure it ? thanks